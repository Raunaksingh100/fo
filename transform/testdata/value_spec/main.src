package main

type A[T] T

type B[T] []T

func main() {
	var a0 A[string] = "foo"
	var _ string = a0
	var a1 A[int] = 42
	var _ int = a1

	var b0 B[string] = []string{"a", "b"}
	var _ []string = b0
}
