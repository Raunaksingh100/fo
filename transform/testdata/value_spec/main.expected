package main

type A interface{}

type B interface{}

func main() {
	var a0 A = "foo"
	var _ string = a0.(string)
	var a1 A = 42
	var _ int = a1.(int)

	var b0 B = []string{"a", "b"}
	var _ []string = b0.([]string)
}
